interfaces/cython/cantera/test/test_models.py

```diff
@@ -0,0 +1,275 @@
+from os.path import join as pjoin
+import os
+
+import numpy as np
+import warnings
+
+try:
+    import ruamel_yaml as yaml
+except ImportError:
+    from ruamel import yaml
+
+import cantera as ct
+from . import utilities
+
+
+class TestModels(utilities.CanteraTest):
+
+    @classmethod
+    def setUpClass(cls):
```

Comment:
I view those tests as going beyond `test_thermo` as they apply to entire solution objects. To clarify, I renamed this to `test_composite`. Arguably, `test_purefluid` could be merged into this more generic case.

Reply:
I felt that `test_thermo` wasn't appropriate as tests really apply to `Solution` objects as a whole. Clarified this as `test_composite` (arguably, it can absorb `test_purefluid`).

Reply:
I felt that `test_thermo` was not appropriate as tests involve entire `Solution` objects. Changing this to `test_composite` (as these tests pertain to `composite.py`).

Reply:
OK, `test_composite` makes more sense.